app/src/main/java/com/boolder/boolder/data/database/DatabaseModules.kt

```diff
@@ -22,3 +23,18 @@ val databaseModule = module {
     single { ProblemRepository(get<BoolderAppDatabase>().problemDao()) }
 
 }
+
+val tickDatabaseModule = module {
+    single {
+        Room.databaseBuilder(
+            androidContext(),
+            TickDatabase::class.java,
+            "tick.db"
+        )
+            .fallbackToDestructiveMigration()
+            .build()
+    }
+
+    single { TickRepository(get<TickDatabase>().tickDao(), get<BoolderAppDatabase>().problemDao(), get<BoolderAppDatabase>().areaDao()) }
+    single { ProblemRepository(get<BoolderAppDatabase>().problemDao()) }
```

Comment:
Any particular reason to instantiate these repositories as singletons?

Reply:
Honestly I was copying the functionality of the `databaseModule`. I can see why we can pass it along instead of making it a singleton. However, Id like to be consistent with both database modules. Is there a reason why this one shouldn't be a singleton and the other should?

Reply:
It was more an open question from me, as it seems that it would also work well by not declaring the repositories as singletons (because singletons are meant to be kept in memory even when navigating to screens where they are not needed)